### PR TITLE
test: rmi should work with images w/o layers

### PR DESCRIPTION
**Pull request by flouthoc**
#### Does this PR introduce a user-facing change?

```release-note
None
```
Test for: https://github.com/containers/common/pull/1651

### CODE REVIEW

**Comment by vrothberg** [comment]
/hold
DO NOT MERGE ... this is vendoring in a fork.

File: test/e2e/manifest_test.go [hunk 1]
```diff
@@ -73,6 +73,48 @@ var _ = Describe("Podman manifest", func() {
 		}
 	})
 
+	It("podman rmi should work after creating empty manifest", func() {
+		session := podmanTest.Podman([]string{"manifest", "create", "foo"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
```
Comment (vrothberg) [inline test/e2e/manifest_test.go:79]: Please use @edsantiago's new `ExitCleanly()` function here and below.

**Comment by flouthoc** [comment]
The test passes on this now.

**Review by vrothberg — COMMENTED** [review]
LGTM but please invert the order of the two commits, otherwise the test won't pass which can cause false positives when bisecting bugs.

**Comment by edsantiago** [comment]
This is a DO NOT MERGE proof-of-concept only, isn't it? Blocking.

**Comment by vrothberg** [comment]
> This is a DO NOT MERGE proof-of-concept only, isn't it? Blocking.

Good to merge now since we're vendoring from c/common and not from the fork anymore.

**Comment by edsantiago** [comment]
Ah I missed that, thanks

/hold cancel

**Comment by edsantiago** [comment]
BUG: this now leaves behind an untagged image: https://api.cirrus-ci.com/v1/artifact/task/5733529070338048/html/sys-podman-fedora-38-root-host-sqlite.log.html#t--00044  (nasty red warnings)

**Pull request merged**